src/schemas/space.json

```diff
@@ -330,20 +330,31 @@
           "type": "object",
           "properties": {
             "maxSupply": {
-              "type": "number",
-              "minimum": 0
+              "type": "integer",
```

Comment:
Why integer and not number?

Reply:
Because it's an integer, and not floating number

Reply:
I see